doc/nrf/known_issues.rst

```diff
@@ -155,6 +155,30 @@ Thread
 
 .. rst-class:: v1-4-0
 
+KRKNWK-6848: Reduced throughput observed.
+  Performance testing for NCP sample shows 20 % decreased throughput with regards with vanilla OpenThread, passing from about 110 kbps to about 89 kbps.
+
+.. rst-class:: v1-4-0
+
+KRKNWK-7885: Throughput is lower when using CC310 nrf_security backend.
+  CC310 nrf_security backend is used by default for nRF52840 boards, but a 5-10 % decreased throughput has been observed when compared with nrf_oberon or vanilla mbedtls backends.
+
+  **Workaround:** Use AES-CCM from the nrf_oberon backend by setting :option:`CONFIG_OBERON_BACKEND` to ``y``, :option:`CONFIG_OBERON_MBEDTLS_AES_C` to ``y``,:option:`CONFIG_OBERON_MBEDTLS_CCM_C` to ``y`` and :option:`CONFIG_CC3XX_MBEDTLS_AES_C` to ``n``.
+
+.. rst-class:: v1-4-0
+
+KRKNWK-7721: The ``RxDestAddrFiltered`` MAC counter is not being updated.
+  This is due to the ``PENDING_EVENT_RX_FAILED`` event not being implemented in Zephyr. Fix is available upstream.
```

Comment:
Can you provide link to the PR with the fix and add it to workaround, as for other cases in known issues?

Reply:
Done.